### ISSUE TITLE
Users can not see when I am streaming

### ISSUE THREAD

**Issue by a09hopper** [MagiChau/ZonBot#1]
Please can you add something like @everyone then the usual text so that people can get a notification.

**Comment by MagiChau** [comment]
No, if people want notifications you should make a separate streams channel and choose to receive notifications of all messages there. Everyone mentions are terribly annoying when done frequently.
